curriculum/challenges/english/14-responsive-web-design-22/learn-html-by-building-a-cat-photo-app/5dfa22d1b521be39a3de7be0.md

```diff
@@ -7,7 +7,9 @@ dashedName: step-12
 
 # --description--
 
-You can see the words `cat photos` underlined next to the image in the preview now. That's your link; feel free to give it a click. In the text of your `p` element, turn the words `cat photos` into a link to `https://freecatphotoapp.com`. When you are done, remove the old anchor tag and text below the paragraph.
+In the previous steps you turned the words `cat photos` into a link by surrounding them with an opening `<a>` tag and closing `</a>` tag. In the preview you can see the words `cat photos` in blue and underlined next to the image. That's your link; feel free to give it a click.
+
+You can turn words inside of a `p` element into a link by surrounding them with an opening `<a>` tag and closing `</a>` tag. Inside the `p` element, turn the words `cat photos` into a link to `https://freecatphotoapp.com`.
```

Comment:
This change clarifies how to nest the new `a` element, but completely omits the instruction to remove the *old* `a` element.